backend/src/main/java/eu/clarin/switchboard/app/config/DataStoreConfig.java

```diff
@@ -99,10 +95,28 @@ public String toString() {
                 .add("\tlocation", location)
                 .add("\teraseAllStorageOnStart", eraseAllStorageOnStart)
                 .add("\tmaxSize", maxSize)
+                .add("\tmaxFiles", maxFiles)
                 .add("\tmaxLifetime", maxLifetime)
                 .add("\tmaxLifetimeUnit", maxLifetimeUnit)
                 .add("\tcleanupPeriod", cleanupPeriod)
                 .add("\tcleanupPeriodUnit", cleanupPeriodUnit)
                 .toString();
     }
+
+    private static long stringToLong(String input) {
```

Comment:
Minor comment but I don't think the name of this method is as descriptive as it could be. Since something somewhat unusual is happening here I think it could do with a bit more (implicit) documentation. I also think it warrants a unit test.

Reply:
Is it better now?

Reply:
I think so!